src/services/ipinfo-client.ts

```diff
@@ -0,0 +1,33 @@
+import { AxiosInstance, AxiosStatic } from 'axios';
+import { ResultWithoutExpectedErrors } from '../doppler-types';
+
+export interface IpinfoResult {
+  countryCode?: string | null;
+}
+
+export interface IpinfoClient {
+  getCountryCode(): Promise<ResultWithoutExpectedErrors<IpinfoResult>>;
```

Comment:
In my opinion, in this case, _ResultWithoutExpectedErrors_ wrapper is not necessary.

```suggestion
  getCountryCode(): Promise<IpinfoResult>;
```

Or even:


```suggestion
  getCountryCode(): Promise<string>;
```